### PR TITLE
Validity range fix; debug info added

### PR DESCRIPTION
**Pull request by peressounko**
ccdb path fixed;Validity range fix; debug info added

### CODE REVIEW

File: Detectors/PHOS/workflow/src/ClusterizerSpec.cxx [hunk 1]
```diff
@@ -43,8 +43,8 @@ void ClusterizerSpec::init(framework::InitContext& ctx)
     ccdbManager.setURL(o2::base::NameConf::getCCDBServer());
     LOG(info) << " set-up CCDB " << o2::base::NameConf::getCCDBServer();
 
-    BadChannelsMap* badMap = ccdbManager.get<o2::phos::BadChannelsMap>("PHS/BadMap");
-    CalibParams* calibParams = ccdbManager.get<o2::phos::CalibParams>("PHS/Calib");
+    BadChannelsMap* badMap = ccdbManager.get<o2::phos::BadChannelsMap>("PHS/Calib/BadMap");
+    CalibParams* calibParams = ccdbManager.get<o2::phos::CalibParams>("PHS/Calib/CalibParams");
```
Comment (shahor02) [inline Detectors/PHOS/workflow/src/ClusterizerSpec.cxx:47]: above you change to this: https://github.com/AliceO2Group/AliceO2/blob/9e546ae0d3a1199026eb910316b7bee97cd84bae/Detectors/PHOS/calib/src/PHOSBadMapCalibDevice.cxx#L137-L145
and here you again use PHS/Calib/BadMap. Could you cross-check?

**Review by shahor02 — COMMENTED** [review]
@peressounko please see the comment below. In general, would be good to gradually switch to DPL CCDB fetcher instead of modifying the code using BasicCCDBManager.

File: Detectors/PHOS/workflow/src/ClusterizerSpec.cxx (hunk 1 above)
Comment (peressounko) [reply Detectors/PHOS/workflow/src/ClusterizerSpec.cxx:47]: Thank you for careful reading. Idea is to store intermediate maps from different methods in PHS/BadMap and manually combine them to final map stored in PHS/Calib/BadMap.

**Comment by peressounko** [comment]
Concerning switching to CCDB fetcher - I see your exchange with Sergey, probably I will switch when everything will be fixed/checked and I  can use his code as an example.

File: Detectors/PHOS/workflow/src/ClusterizerSpec.cxx (hunk 1 above)
Comment (shahor02) [reply Detectors/PHOS/workflow/src/ClusterizerSpec.cxx:47]: Why manual (which we want to avoid). How often will you do this?

Comment (peressounko) [reply Detectors/PHOS/workflow/src/ClusterizerSpec.cxx:47]: Ingredients of bad map will be run automatically. Combination will be done once per modification of electronics ~ as in Run2 per period or few periods.

**Comment by peressounko** [comment]
Can not find what is the problem in build/O2/o2-dataflow-cs8?

**Review by shahor02 — APPROVED** [review]

**Pull request merged**